index.js

```diff
@@ -1,6 +1,9 @@
 import React from "react";
 import { registerRootComponent } from "expo";
 import App from "./App";
+import axios from "axios";
+
+axios.defaults.baseURL = "http://localhost:3000/api/v1";
```

Comment:
Change this to your deployed url so that you are connecting it to the backend when demonstrating.

Reply:
We tried to use the .env file but it wasn't